code/game/machinery/telecomms/broadcasting.dm

```diff
@@ -174,14 +174,13 @@
 	// Always call this on the virtualspeaker to avoid issues.
 	var/spans = data["spans"]
 	var/list/message_mods = data["mods"]
-	var/rendered = virt.compose_message(virt, language, message, frequency, data["frequency_name"], data["frequency_color"], spans)
```

Comment:
Core radio code composed a redundant message that was never used anywhere - that's free perf baby